packs/vsphere/actions/vmwarelib/actions.py

```diff
@@ -51,8 +57,8 @@ def _connect(self, vsphere):
                                       port=connection['port'],
                                       user=connection['user'],
                                       pwd=connection['passwd'])
-        except:
-            raise Exception("Unable to connect to vsphere.")
+        except Exception as e:
```

Comment:
You could also remove Exception all together, this way original (and probably more specific) exception is throw.